modules/ppcp-blocks/src/BlocksModule.php

```diff
@@ -69,8 +69,8 @@ function () {
 			function( PaymentMethodRegistry $payment_method_registry ) use ( $c ): void {
 				$payment_method_registry->register( $c->get( 'blocks.method' ) );
 
-				// Include ACDC in the Block Checkout only in case Axo doesn't exist or is not available.
-				if ( ! $c->has( 'axoblock.available' ) || ! $c->get( 'axoblock.available' ) ) {
+				// Include ACDC in the Block Checkout only in case Axo doesn't exist or is not available or the user is logged in.
+				if ( ! $c->has( 'axoblock.available' ) || ! $c->get( 'axoblock.available' ) || is_user_logged_in() ) {
```

Comment:
I think it makes more sense to move the `is_user_logged_in()` check to the `'axoblock.available'` service:

instead of hard-coded return true, use `return ! is_user_logged_in()`

Reply:
@danieldudzic and me came to the conclusion that the `<gateway>.available` service always checks for a merchant-level-availability. Performing user-specific checks should be done outside that service, that's why we'll go with this proposed condition